include/nanobind/stl/bind_map.h

```diff
@@ -0,0 +1,196 @@
+/*
+    nanobind/stl/bind_map.h: nb::bind_map()
+
+    This implementation is a port from pybind11 with minimal adjustments.
```

Comment:
I think this line can go away, the implementation is quite different at this point.